### PR TITLE
docs: move to readthedocs.io

### PR DESCRIPTION
**Pull request by tiborsimko**
Signed-off-by: Tibor Simko <tibor.simko@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
LGTM

File: docs/testing.rst [hunk 1]
```diff
@@ -79,4 +79,4 @@ that was written in it without having to create temporary files with
 :py:mod:`tempfile`.
 
 
-.. _PyHamcrest: http://pythonhosted.org//PyHamcrest/
+.. _PyHamcrest: http:///PyHamcrest.readthedocs.io/
```
Comment (jirikuncar) [inline docs/testing.rst]: hups, tripple `///`

**Comment by jirikuncar** [comment]
LGTM

File: docs/testing.rst (hunk 1 above)
Comment (tiborsimko) [reply docs/testing.rst]: (Thanks, my script did not look at cases or multiple slashes... Something to remember for next time.)

**Pull request merged**